docs/alerts/monitors/alert-response.md

```diff
@@ -229,18 +229,48 @@ A _Monitor_ creates an _Alert_. Using the options below, you're subscribing to a
 :::
 
 
-#### From your Alerts list
+### From your Alerts list
 * Right-click on a row item > click **Subscribe**
 * Hover your mouse over a row, click the three-dot kebab menu > select **Subscribe**
 * Single-click on a row item > on the opened Alert page, click the three-dot kebab menu > **Subscribe to Monitor**
 
 
-#### From your Monitors list
+### From your Monitors list
 * Right-click on a row item > click **Subscribe**
 * Hover your mouse over a row > click the three-dot kebab menu > click **Subscribe**
 * Single-click on a row item > in the side panel (Monitor Details), click **More Actions** > **Subscribe**
 
 
+## Muting Schedules
```

Comment:
@kimsauce  Muting Schedules is a new Tab in Sumo, and we should have a new Page for it on the https://help.sumologic.com/docs/alerts/monitors/ 

It should certainly not go under Alert Response Page.